sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -24,5 +24,7 @@ export enum AppRoutes {
   InstitutionManageUsers = "manage-users",
   DisabledUser = "login/disabled-user",
   StudentApplication = "application-form/:id?/:selectedForm",
+  StudentApplicationSummary = "my-application-summary",
```

Comment:
It seems that these routes are not aligned: StudentApplication, StudentApplicationSummary, and StudentEditApplication.
StudentEditApplication is just a placeholder for now, right?

Reply:
as discussed closing this thread